### PR TITLE
OCPBUGS#9375: Add generic ephemeral volumes

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-9375

Link to docs preview: https://57529--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints.html#authorization-controlling-volumes_configuring-internal-oauth

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by gnufied** [comment]
cc @dobsonj

File: modules/security-context-constraints-about.adoc [hunk 1]
```diff
@@ -330,6 +330,7 @@ sources that are defined when creating a volume:
 * link:https://kubernetes.io/docs/concepts/storage/volumes/#flexvolume[`flexVolume`]
 * link:https://kubernetes.io/docs/concepts/storage/volumes/#flocker[`flocker`]
 * link:https://kubernetes.io/docs/concepts/storage/volumes/#gcepersistentdisk[`gcePersistentDisk`]
+* link:https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes[Generic ephemeral volumes]
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/security-context-constraints-about.adoc]: Addressed

**Comment by duanwei33** [comment]
@Phaow Could you please take a look?

**Comment by Phaow** [comment]
> @Phaow Could you please take a look?
Sure.

**Comment by Phaow** [comment]
LGTM

**Comment by Phaow** [comment]
@gnufied @dobsonj BTW, from my understanding, we should also add the [`csi` volume](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes) in the SCC Controlling volumes list for 4.13+ since csi-ephemeral-volumes GA in 4.13 and we already add it to all preset SCC, right?

**Comment by dobsonj** [comment]
> @gnufied @dobsonj BTW, from my understanding, we should also add the [`csi` volume](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes) in the SCC Controlling volumes list for 4.13+ since csi-ephemeral-volumes GA in 4.13 and we already add it to all preset SCC, right?

@Phaow yes that's right, `ephemeral` volumes were added to SCC's [here](https://github.com/openshift/cluster-kube-apiserver-operator/pull/1380), and `csi` volumes were added to SCC's [here](https://github.com/openshift/cluster-kube-apiserver-operator/pull/1434).

**Comment by Phaow** [comment]
@dobsonj Thanks for the details and doulbe confirm! Seems we also need to add doc for csi SCC in the chapter.

**Comment by Phaow** [comment]
Hi @dobsonj , IMO, the csi inline volumes is TP on below 4.13 versions, but if customer want to use it also need a custom scc add the csi volume, but it seems the TP support from very earily, maybe only added for 4.13 is acceptable. But I'm not sure we need a separate doc bug track it. WDYT?

**Comment by Phaow** [comment]
@dobsonj Got it. Thanks for double confirm!

**Comment by xenolinux** [comment]
For tracking the change related to csi volumes I created a PR here https://github.com/openshift/openshift-docs/pull/58313. Thanks

**Comment by Phaow** [comment]
LGTM

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by kalexand-rh** [comment]
/remove-lifecycle stale

@xenolinux, can this one be merged?

**Comment by duanwei33** [comment]
I think this is still needed and important to align with our code, could it be merged? 
@xenolinux @kalexand-rh

**Comment by xenolinux** [comment]
@duanwei33 @kalexand-rh Sorry I missed updates on this PR. I will get it merged asap. Aplogies

**Pull request merged**